src/function-forms/error-no-strict/async-arrow-function.template

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2017 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/expressions/async-arrow-function/
+name: async arrow function expression in sloppy code
+esid: sec-async-arrow-function-definitions
+flags: [noStrict]
+info: |
+  14.7 Async Arrow Function Definitions
+
+  AsyncArrowFunction :
+    ...
+    CoverCallExpressionAndAsyncArrowHead => AsyncConciseBody
+
+  AsyncConciseBody :
+    { AsyncFunctionBody }
+
+  ...
+
+  Supplemental Syntax
+
+  When processing an instance of the production AsyncArrowFunction :
+  CoverCallExpressionAndAsyncArrowHead => AsyncConciseBody the interpretation of
+  CoverCallExpressionAndAsyncArrowHead is refined using the following grammar:
+
+  AsyncArrowHead :
+    async ArrowFormalParameters
+flags: [async]
```

Comment:
The two `flag:` sections should be combined. The first `noStrict` flags section is being ignored.